packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.test.tsx

```diff
@@ -109,23 +109,33 @@ describe('AccountRecoveryResetPassword page', () => {
         </LocationProvider>
       </AppContext.Provider>
     );
+
+    // let's page settle and avoid 'act' warning messages
+    // await screen.findByRole('heading', { level: 1 });
   }
 
-  function clickResetPassword() {
-    screen.getByRole('button', { name: 'Reset password' }).click();
+  async function clickResetPassword() {
+    const button = await screen.findByRole('button', {
+      name: 'Reset password',
+    });
+    await act(async () => {
+      button.click();
+    });
   }
 
-  function clickReceiveNewLink() {
-    screen
-      .getByRole('button', {
-        name: 'Receive new link',
-      })
-      .click();
+  async function clickReceiveNewLink() {
+    const button = await screen.findByRole('button', {
+      name: 'Receive new link',
+    });
+
+    await act(async () => {
+      button.click();
+    });
```

Comment:
I _think_ you can just use `fireEvent.click(button);` here.

Reply:
Happy to change it, but why is that preferred? Are you implying the await act isn't needed? 

BTW, this was changed because before doing this there was a react warning.